src/symbolic/symbolic_choice.mli

```diff
@@ -13,6 +13,8 @@ include
      and type 'a run_result = ('a eval * Thread.t) Seq.t
      and module V := Symbolic_value
 
+val with_new_symbol : Smtml.Ty.t -> (Smtml.Symbol.t -> 'b) -> 'b t
```

Comment:
I'm not sure if I should add this to `Choice_intf.Complete` instead?

Reply:
It's fine to have it there for now. :)